android/src/main/java/com/middlewarereactnative/MiddlewareReactNativeModule.java

```diff
@@ -242,7 +243,7 @@ private ReactSpanProperties propertiesFromMap(SpanMapReader mapReader) {
 
     final ReadableArray readerEvents = mapReader.getEvents();
     final List<EventData> newEvents = new ArrayList<>();
-    for(int index = 0; index < readerEvents.size(); index++) {
+    for (int index = 0; index < readerEvents.size(); index++) {
```

Comment:
We can use for-each loop.

Reply:
Since, we are using ReadableArray to use foreach we will have to convert to `toArrayList()` which in turn will run loop inside storing objects in List. Hence, using normal loop here.

Reply:
okay, got it.